ac_termination_protection/variables.tf

```diff
@@ -0,0 +1,65 @@
+variable "name" {
+  type        = string
+  description = "The name of the resource"
+}
+
+variable "namespace" {
+  type        = string
+  description = "The namespace of the resource"
+}
+
+variable "docker_image" {
+  description = "The docker image to use for the container"
+
+  type = object({
+    name = string
+    args = list(string)
+  })
+
+  default = {
+    name = "vutoff/k8s-termination-protection:v0.1"
+    args = [
+      "-C",
+      "puma.rb"
+    ]
```

Comment:
These should be the default of the Docker image, and not present here. The default should be a blank array.